etc/Makefile

```diff
@@ -44,7 +44,7 @@ TOREE_LAUNCHER_FILES:=$(shell find kernel-launchers/scala/toree-launcher/src -ty
     # Seed the build tree with initial files
 	cp -r kernelspecs ../build
     # Distribute language and config-sensitive files.
-	@echo ../build/kernelspecs/{spark_,}python_* | xargs -t -n 1 cp -r kernel-launchers/python/scripts
+	@echo ../build/kernelspecs/{spark_,dask_}python_* | xargs -t -n 1 cp -r kernel-launchers/python/scripts
```

Comment:
We need to preserve the trailing comma, otherwise directories beginning with `python_` won't get their launchers.

Reply:
Thanks, fixed.